Cargo.toml

```diff
@@ -94,3 +94,18 @@ serde = ["serde_crate", "serde_with", "serde_yaml", "bp-std/serde", "bp-wallet/s
 
 [package.metadata.docs.rs]
 features = ["all"]
+
+[patch.crates-io]
+strict_types = { git = "https://github.com/strict-types/strict-types", branch = "master" }
```

Comment:
As reported here https://github.com/RGB-WG/rgb-std/pull/180#pullrequestreview-1991388328 rgb-std is not compiling with these patches. If you run `cargo update` on this repo you'll see the same error that I reported. It seems that in the `Cargo.lock` file of this repo strict_types was set to `master#2350162c`, a commit that is not directly referenced by any branch and that is no longer in master history. Since I've seen that commit is part of 2 branches (`develop` and `reconstruct`) I've tried both and I've found out that I can compile rgb-lib using the `reconstruct` branch. So either we merge that commit on master or we change the patched branch here.

Reply:
Fixed to reconstruct. Pushed